bin/run_crab_unique_users.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+# With this environment, It works in lxplus7, it can requiere modifications to run elsewhere.
```

Comment:
We need usage message here as well, it should also outline what is required to run the script, i.e. cvmfs, java, etc.

Reply:
I added a help file, should we reference it from here?

Reply:
yes